app/src/main/java/com/example/memeapp/presentation/mvp/presenter/MemeAppPresenter.kt

```diff
@@ -29,4 +29,13 @@ class MemeAppPresenter(
     override fun onMemeClicked(memeId: Int) {
         view.showMemeInfo(memeId)
     }
+
+    override fun getMemesFromDataBase() {
+        view.cleanRecycler()
+        view.showProgressBar(true)
+        model.loadMemesFromDataBase().apply {
+            view.showData(this)
+            view.showProgressBar(false)
+        }
```

Comment:
Why you are using a scope function here, if you are not using the model scope?

Reply:
now its no more, thx